### ISSUE TITLE
 "Air.mix" properties calculation

### ISSUE THREAD

**Issue by vitcaro** [CoolProp/CoolProp#1338]
"Air.mix" calculation over 67°C (no matter the pressure value) gives an error message.

Example (Excel wrapper):
=PropsSI("D";"T";70+273,15;"P";101325;"Air.mix")

Message:
One stationary point (not good) for T=343.15,p=101325,z=[ 0.7812, 0.0092, 0.2096 ] : PropsSI("D","T",343.15,"P",101325,"Air.mix")

Same problem using (in place of  "Air.mix") the mixture: 
"Nitrogen[.7809]&Oxygen[.2095]&Argon[.0093]&CO2[.00039]" 
This is the mixture (for dry air) in the file (part of the Excel 
wrapper documentation) "TestExcel.xlsx", sheet "Sample Calcs".

**Comment by ibell** [comment]
Yes, I know I need to come back in and do some more testing of the PT flashes for mixtures.  It's so incredibly painful.... You'll have to use REFPROP for this case for now until I have a chance to dig into this.

**Comment by Joao-Souza** [comment]
Hey, I've been having that same issue here. Have you had that fixed yet?

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
